.gitignore

```diff
@@ -12,6 +12,7 @@
 /build
 
 # misc
+.env
```

Comment:
Does `.env.local` work?

Reply:
The .env.local line here doesn't hide the .env file, if that's what you're asking.

Reply:
Correct, but are we able to put the environment variables we need into a `.env.local` file instead of adding a `.env` rule to the gitignore?

Reply:
It does, LOL. I'll get rid of the .env and merge